book/src/dev/continuous-integration.md

```diff
@@ -33,10 +33,27 @@ We try to use Mergify as much as we can, so all PRs get consistent checks.
 Some PRs don't use Mergify:
 - Mergify config updates
 - Admin merges, which happen when there are multiple failures on the `main` branch
-- Manual merges
+  (these are disabled by our branch protection rules, but admins can remove the "don't allow bypassing these rules" setting)
+- Manual merges (these are usually disabled by our branch protection rules)
 
 We use workflow conditions to skip some checks on PRs, Mergify, or the `main` branch.
-For example, some workflow changes skip Rust code checks.
+For example, some workflow changes skip Rust code checks. When a workflow can skip a check, we need to create a patch workflow
+with an empty job with the same name. This lets the branch protection rules pass when the job is skipped. In Zebra, we name these
```

Comment:
@gustavovalverde I tried to find some GitHub or Stack Overflow documentation for patch workflows, but I couldn't track it down. Do you know where it is?

I'd like to link to it here.

Reply:
might be this one: https://github.com/orgs/community/discussions/13690